docroot/modules/custom/mass_content/src/Field/RelatedNodes.php

```diff
@@ -52,9 +52,9 @@ class RelatedNodes extends EntityReferenceFieldItemList {
    * {@inheritdoc}
    */
   public function computeValue() {
-    $nids_fields = $this->fields();
     $nids_links = $this->links();
-    $nids_total = array_merge($nids_fields, $nids_links);
+    $nids_fields = $this->fields();
+    $nids_total = array_diff($nids_links, $nids_fields);
```

Comment:
@weitzman In the old version of this, we had a condition in place that filtered out nodes that were manually set against this content. It was done in a convoluted way where we created an array of NIDs in computeValue() and passed those values to query(). I think changing this to an array_diff() replicates that behavior, but I didn't really notice a difference in the TOC with or without this in place.